deb/openmediavault/var/www/openmediavault/js/omv/Rpc.js

```diff
@@ -167,7 +168,11 @@ Ext.define("OMV.Rpc", {
 				me.setDisabled(true);
 				// Display a dialog forcing user to click 'OK' to reload
 				// the page.
-				OMV.MessageBox.guru({
+				if (this.guru)
+				{
+					this.guru.destroy();
+				}
+				this.guru = OMV.MessageBox.guru({
```

Comment:
The problem is when the user session expires but there is a recurring query:
The recurring query is still active, and a new guru meditation window will be created each time there is an (unsuccessful) reply to the query.